pkg/DWSUtility.py

```diff
@@ -368,14 +368,14 @@ def do_resource_list(self):
 
             try:
                 crd_obj = self.dws.get_custom_resource_definition(crd)
-            except:
+            except Exception:
                 continue
             printer_cols = self.dws.get_crd_printer_columns(crd_obj)
```

Comment:
Shouldn't this be above in the try/except block since `crd_ojb` could be none here?